README.md
# Wonderkamer

Source and content for Wonderkamer website.

## Development

For local development see [CONTRIBUTING.md](./CONTRIBUTING.md).

Push or merge into the **develop** branch and a github action will run to build and test and deploy the resulting build 
directory to [dev.wonderkamer.com](https://dev.wonderkamer.com). It is password protected.

## Publishing

Push or merge into the **master** branch and a github action will run to build and test and deploy the resulting build 
directory to [wonderkamer.com](https://wonderkamer.com).


CHANGELOG.md
# Changelog

All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.

### [1.2.4](https://github.com/Wonderkamer/website/compare/1.2.3...1.2.4) (2021-07-13)


### Content Changes

* leden toevoegen, bio's etc ([e2749e5](https://github.com/Wonderkamer/website/commits/e2749e5fd7667e366ee1d4e58cf9e149e608d9c3))
* leden toevoegen, bio's etc ([267365c](https://github.com/Wonderkamer/website/commits/267365ca9dddae5215f0b75b7fcfcb7793a136bd))
* tekstuele en photo update ([88f0537](https://github.com/Wonderkamer/website/commits/88f053784b3a9c4ddd1fccbe214326524a1e6fc7))


### Refactoring

* route reload ([750019f](https://github.com/Wonderkamer/website/commits/750019fd53184329dd1f22a4a499e483671389e3))

### [1.2.3](https://github.com/Wonderkamer/website/compare/1.2.2...1.2.3) (2021-04-19)


### Content Changes

* lid verwijderen en kopij (internetsnelheid) aanpassen ([32cd2cc](https://github.com/Wonderkamer/website/commits/32cd2cc66f9b0f793a51e9e79c45ac9cb7331992))

### [1.2.2](https://github.com/Wonderkamer/website/compare/1.2.1...1.2.2) (2021-03-22)


### Content Changes

* member changes ([54cde89](https://github.com/Wonderkamer/website/commits/54cde8902608b5fd6b70829b1eeaf4dea557b0ab))
* tekst wijziging ([2d4c020](https://github.com/Wonderkamer/website/commits/2d4c0204ef583543bb9d74a4b85ff6ac8a1a2a0f))
* tekst wijziging ([c2163dc](https://github.com/Wonderkamer/website/commits/c2163dca68cad7166a94f38fa2d6818458f352dd))


### Miscellaneous

* maps update ([a3c03bf](https://github.com/Wonderkamer/website/commits/a3c03bf714901f8dbbcb686d7287f9429cbccbb6))
* replace non working library ([3d4215c](https://github.com/Wonderkamer/website/commits/3d4215c44a2405311ca10e1e54c39ca3f2dbc4e7))
* updated packages ([359cfe5](https://github.com/Wonderkamer/website/commits/359cfe5a4db5004f3f7f2be63058e84d5618fce3))

### [1.2.1](https://github.com/Wonderkamer/website/compare/1.2.0...1.2.1) (2021-03-18)


### Content Changes

* rename leden a beetje ([d4bf5d4](https://github.com/Wonderkamer/website/commits/d4bf5d470f7caa890793e59d9673be2674d09e68))

## 1.2.0 (2021-03-18)


### Features

* adds a tracking param 'source' ([c09df1b](https://github.com/Wonderkamer/website/commits/c09df1b1788877d1e659f052904912ba71973620))


### Miscellaneous

* adds tooling to release, lint staged files and commit messages ([40b2ee5](https://github.com/Wonderkamer/website/commits/40b2ee5b02c1d813f4e270932691f78eccb116e1))
* reformatted ([8b8e7ce](https://github.com/Wonderkamer/website/commits/8b8e7cecadc7756ecabaeb62de69d51f32bf39b2))
* updated the caniuse browserlist db ([7e0f2e5](https://github.com/Wonderkamer/website/commits/7e0f2e504ca2286a195b5e332204be5615df6781))
* upgraded various packages ([51626fb](https://github.com/Wonderkamer/website/commits/51626fbad37695260e809748a5eaeedce3564e52))


### Content Changes

* aanpassingen leden ([94ec886](https://github.com/Wonderkamer/website/commits/94ec8863b3a9837a985563c4df5bc7407693c2bc))


mirage/config.js
import ENV from 'wonderkamer/config/environment';

export default function () {
  this.get('/members/:id', (schema, request) => {
    return schema.members.find(request.params.id);
  });
  this.get('/members', (schema, request) => {
    if (request.queryParams.slug) {
      return schema.members.where(request.queryParams);
    }

    return schema.members.all();
  });

  this.get('occupancy.php', (schema, request) => {
    const dataPoints = [
      { on: '2020-08-06T15:15:03+00:00', occupancy: 1 },
      { on: '2020-08-06T15:19:56+00:00', occupancy: 1 },
      { on: '2020-08-06T15:31:01+00:00', occupancy: 2 },
      { on: '2020-08-06T15:34:47+00:00', occupancy: 1 },
      { on: '2020-08-06T15:43:42+00:00', occupancy: 1 },
      { on: '2020-08-06T15:45:16+00:00', occupancy: 1 },
      { on: '2020-08-06T16:00:20+00:00', occupancy: 1 },
      { on: '2020-08-06T16:15:23+00:00', occupancy: 1 },
      { on: '2020-08-10T15:33:35+00:00', occupancy: 3 },
      { on: '2020-08-10T15:34:08+00:00', occupancy: 3 },
      { on: '2020-08-10T15:34:40+00:00', occupancy: 3 },
      { on: '2020-08-10T15:35:11+00:00', occupancy: 3 },
      { on: '2020-08-10T15:35:44+00:00', occupancy: 3 },
      { on: '2020-08-10T15:36:16+00:00', occupancy: 3 },
      { on: '2020-08-10T15:36:48+00:00', occupancy: 3 },
      { on: '2020-08-10T15:37:20+00:00', occupancy: 3 },
      { on: '2020-08-10T15:37:52+00:00', occupancy: 3 },
      { on: '2020-08-10T15:38:24+00:00', occupancy: 3 },
      { on: '2020-08-10T15:41:07+00:00', occupancy: 3 },
      { on: '2020-08-10T15:41:39+00:00', occupancy: 3 },
      { on: '2020-08-10T15:42:11+00:00', occupancy: 3 },
      { on: '2020-08-10T15:42:43+00:00', occupancy: 3 },
      { on: '2020-08-10T15:43:21+00:00', occupancy: 3 },
      { on: '2020-08-10T15:44:22+00:00', occupancy: 3 },
      { on: '2020-08-10T15:45:21+00:00', occupancy: 3 },
      { on: '2020-08-10T15:45:53+00:00', occupancy: 3 },
      { on: '2020-08-10T16:00:30+00:00', occupancy: 3 },
      { on: '2020-08-10T16:15:03+00:00', occupancy: 3 },
      { on: '2020-08-10T16:30:06+00:00', occupancy: 2 },
      { on: '2020-08-10T16:45:09+00:00', occupancy: 2 },
      { on: '2020-08-10T17:00:12+00:00', occupancy: 3 },
      { on: '2020-08-10T17:15:16+00:00', occupancy: 3 },
      { on: '2020-08-10T17:30:19+00:00', occupancy: 3 },
      { on: '2020-08-10T17:45:22+00:00', occupancy: 2 },
      { on: '2020-08-10T18:00:26+00:00', occupancy: 2 },
      { on: '2020-08-10T18:15:29+00:00', occupancy: 2 },
      { on: '2020-08-10T18:30:02+00:00', occupancy: 1 },
      { on: '2020-08-10T18:45:05+00:00', occupancy: 2 },
      { on: '2020-08-10T19:00:10+00:00', occupancy: 2 },
      { on: '2020-08-10T19:15:13+00:00', occupancy: 2 },
      { on: '2020-08-10T19:30:17+00:00', occupancy: 2 },
      { on: '2020-08-10T19:45:20+00:00', occupancy: 2 },
      { on: '2020-08-10T20:00:24+00:00', occupancy: 2 },
      { on: '2020-08-10T20:15:27+00:00', occupancy: 2 },
      { on: '2020-08-10T20:30:30+00:00', occupancy: 2 },
      { on: '2020-08-10T20:45:03+00:00', occupancy: 2 },
      { on: '2020-08-10T21:00:06+00:00', occupancy: 2 },
      { on: '2020-08-10T21:15:10+00:00', occupancy: 2 },
      { on: '2020-08-10T21:30:13+00:00', occupancy: 2 },
      { on: '2020-08-10T21:45:16+00:00', occupancy: 1 },
      { on: '2020-08-10T22:00:19+00:00', occupancy: 1 },
      { on: '2020-08-10T22:15:22+00:00', occupancy: 1 },
      { on: '2020-08-10T22:30:26+00:00', occupancy: 1 },
      { on: '2020-08-10T22:45:29+00:00', occupancy: 1 },
      { on: '2020-08-10T23:00:32+00:00', occupancy: 1 },
      { on: '2020-08-10T23:15:05+00:00', occupancy: 1 },
      { on: '2020-08-10T23:30:08+00:00', occupancy: 1 },
      { on: '2020-08-10T23:45:11+00:00', occupancy: 1 },
      { on: '2020-08-11T00:00:14+00:00', occupancy: 1 },
      { on: '2020-08-11T00:15:17+00:00', occupancy: 1 },
      { on: '2020-08-11T00:30:21+00:00', occupancy: 1 },
      { on: '2020-08-11T00:45:24+00:00', occupancy: 1 },
      { on: '2020-08-11T01:00:27+00:00', occupancy: 1 },
      { on: '2020-08-11T01:15:30+00:00', occupancy: 1 },
      { on: '2020-08-11T01:30:03+00:00', occupancy: 1 },
      { on: '2020-08-11T01:45:06+00:00', occupancy: 1 },
      { on: '2020-08-11T02:00:11+00:00', occupancy: 1 },
      { on: '2020-08-11T02:15:14+00:00', occupancy: 1 },
      { on: '2020-08-11T02:30:17+00:00', occupancy: 1 },
      { on: '2020-08-11T02:45:20+00:00', occupancy: 1 },
      { on: '2020-08-11T03:00:23+00:00', occupancy: 1 },
      { on: '2020-08-11T03:15:26+00:00', occupancy: 1 },
      { on: '2020-08-11T03:30:29+00:00', occupancy: 1 },
      { on: '2020-08-11T03:45:02+00:00', occupancy: 1 },
      { on: '2020-08-11T04:00:05+00:00', occupancy: 1 },
      { on: '2020-08-11T04:15:08+00:00', occupancy: 1 },
      { on: '2020-08-11T04:30:11+00:00', occupancy: 1 },
      { on: '2020-08-11T04:45:14+00:00', occupancy: 1 },
      { on: '2020-08-11T05:00:18+00:00', occupancy: 1 },
      { on: '2020-08-11T05:15:21+00:00', occupancy: 1 },
      { on: '2020-08-11T05:30:24+00:00', occupancy: 1 },
      { on: '2020-08-11T05:45:27+00:00', occupancy: 1 },
      { on: '2020-08-11T06:00:30+00:00', occupancy: 1 },
      { on: '2020-08-11T06:15:03+00:00', occupancy: 1 },
      { on: '2020-08-11T06:30:06+00:00', occupancy: 1 },
      { on: '2020-08-11T06:45:09+00:00', occupancy: 1 },
      { on: '2020-08-11T07:00:12+00:00', occupancy: 1 },
      { on: '2020-08-11T07:15:15+00:00', occupancy: 1 },
      { on: '2020-08-11T07:30:18+00:00', occupancy: 1 },
      { on: '2020-08-11T07:45:21+00:00', occupancy: 1 },
      { on: '2020-08-11T08:00:25+00:00', occupancy: 1 },
      { on: '2020-08-11T08:15:28+00:00', occupancy: 1 },
      { on: '2020-08-11T08:30:31+00:00', occupancy: 1 },
      { on: '2020-08-11T08:45:04+00:00', occupancy: 2 },
      { on: '2020-08-11T09:00:07+00:00', occupancy: 2 },
      { on: '2020-08-11T09:15:11+00:00', occupancy: 2 },
      { on: '2020-08-11T09:30:14+00:00', occupancy: 2 },
      { on: '2020-08-11T09:45:17+00:00', occupancy: 2 },
      { on: '2020-08-11T10:00:21+00:00', occupancy: 2 },
      { on: '2020-08-11T10:15:24+00:00', occupancy: 2 },
      { on: '2020-08-11T10:30:27+00:00', occupancy: 2 },
      { on: '2020-08-11T10:45:30+00:00', occupancy: 2 },
      { on: '2020-08-11T11:00:04+00:00', occupancy: 2 },
      { on: '2020-08-11T11:15:07+00:00', occupancy: 1 },
      { on: '2020-08-11T11:30:11+00:00', occupancy: 1 },
      { on: '2020-08-11T11:45:14+00:00', occupancy: 1 },
      { on: '2020-08-11T12:00:17+00:00', occupancy: 1 },
      { on: '2020-08-11T12:15:20+00:00', occupancy: 1 },
      { on: '2020-08-11T12:30:23+00:00', occupancy: 1 },
      { on: '2020-08-11T12:45:26+00:00', occupancy: 1 },
      { on: '2020-08-11T13:00:30+00:00', occupancy: 1 },
      { on: '2020-08-11T13:15:03+00:00', occupancy: 7 },
    ];

    return JSON.stringify(dataPoints);
  });

  this.passthrough('/members/**');
  this.passthrough('/sections/**');
  this.passthrough('/contact-form.php');

  if (ENV['environment'] === 'production') {
    this.passthrough('/occupancy.php');
  }

  if (ENV['environment'] === 'testing') {
    this.passthrough('/occupancy.php');
  }
}
